components/prism-naniscript.js

```diff
@@ -0,0 +1,172 @@
+(function (Prism) {
+
+	function expressionDef(isMultiline) {
+		return RegExp(/\{[^\r\n\[\]{}]*\}/, isMultiline ? 'm': '');
+	};
+
+	var params = {
+		'quoted-string': {
+			pattern: /"(?:[^"\\]|\\.)*"/,
+			alias: 'operator'
+		},
+		'command-param-id': {
+			pattern: /(\s)\w+:/,
+			lookbehind: true,
+			alias: 'property'
+		},
+		'command-param-value': [
+			{
+				pattern: expressionDef(false),
+				alias: 'selector',
+			},
+			{
+				pattern: /([\t ])\S+/,
+				lookbehind: true,
+				greedy: true,
+				alias: 'operator',
+			},
+			{
+				pattern: /\S(?:.*\S)?/,
+				alias: 'operator',
+			}
+		]
+	};
+
+	Prism.languages.naniscript = {
+		// ; ...
+		'comment': {
+			pattern: /^([\t ]*);.*/m,
+			lookbehind: true,
+		},
+		// > ...
+		// Define is a control line starting with '>' followed by a word, a space and a text.
+		'define': {
+			pattern: /^>.+/m,
+			alias: 'tag',
+			inside: {
+				'value': {
+					pattern: /(^>\w+[\t ]+)(?!\s)[^{}\r\n]+/,
+					lookbehind: true,
+					alias: 'operator'
+				},
+				'key': {
+					pattern: /(^>)\w+/,
+					lookbehind: true,
+				}
+			}
+		},
+		// # ...
+		'label': {
+			pattern: /^([\t ]*)#[\t ]*\w+[\t ]?$/m,
```

Comment:
This still has to be changed.

```suggestion
			pattern: /^([\t ]*)#[\t ]*\w+[\t ]*$/m,
```